config/ConnectorConfig.json

```diff
@@ -5,7 +5,8 @@
       "Credentials":"UEktU2VydmVyOk1hbmNoZXN0ZXIxMjMh"
    },
    "eWONConfig":{
-      "CertificatePath":"/usr/Certificates"
+      "CertificatePath":"/usr/Certificates",

```

Comment:
This config file should probably be removed. The release script will include the config file in the starting files directory.
https://github.com/hms-networks/flexy-osisoft-connector/blob/main/starting-files/ConnectorConfig.json

Reply:
The entire ConnectorConfig.json file? Or this line of code? Or the the change to this file?

Reply:
ok.